PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -12,16 +8,3 @@ Description here
 - [ ] Bug fix (non-breaking change that fixes an issue)
 - [ ] New feature (non-breaking change that adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-
-## How to test
-
-1. A short list
-1. To tell the reviewer
-1. How to test your code
-
-## Assumptions
-
-If any of the following are checked, please explain in the PR summary.
-
-- [ ] This change affects _Security_
```

Comment:
I think this is a compliance requirement, whether we use it or not. This is a signal that we should have the security "team" review the PR.